com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -459,7 +459,7 @@ public static bool DrawHeaderToggle(string title, SerializedProperty group, Seri
         /// <param name="hasMoreOptions">Delegate saying if we have MoreOptions</param>
         /// <param name="toggleMoreOptions">Callback called when the MoreOptions is toggled</param>
         /// <returns>return the state of the foldout header</returns>
-        public static bool DrawHeaderToggle(GUIContent title, SerializedProperty group, SerializedProperty activeField, Action<Vector2> contextAction = null, Func<bool> hasMoreOptions = null, Action toggleMoreOptions = null)
+        public static bool DrawHeaderToggle(GUIContent title, SerializedProperty group, SerializedProperty activeField, string documentationURL, Action<Vector2> contextAction = null, Func<bool> hasMoreOptions = null, Action toggleMoreOptions = null)
```

Comment:
You are changing the signature of this method and will break everyone that is using this. 
Just load URP and it breaks.